.github/workflows/ci.yml

```diff
@@ -57,6 +57,7 @@ jobs:
           - aarch64-unknown-linux-musl
           - mips-unknown-linux-musl
           - mipsel-unknown-linux-musl
+          - x86_64-unknown-linux-gnu
```

Comment:
Rename to `x86_64-debian-gnu` ?

Reply:
fixed in https://github.com/helium/gateway-rs/pull/415/commits/6ec9e0b9a1cc0b28121ed75ac9c7e5881ef9522b